src/textual/css/stylesheet.py

```diff
@@ -124,10 +124,17 @@ def apply(self, node: DOMNode) -> None:
 
         get_first_item = itemgetter(0)
 
+        log("")
+        log(node)
+        for key, attributes in rule_attributes.items():
```

Comment:
Should we maybe add some more context as to what these log messages mean?

Reply:
Those are purely for debugging. Forgive those for the moment. Will clear them in a future update.